### ISSUE TITLE
How can I add new blocks?

### ISSUE THREAD

**Issue by leosfont** [dotswan/filament-grapesjs-v3#1]
How can I add new blocks and disable the builder's tailwind components? they have old tailwind html..

**Comment by thomas-alrek** [comment]
Wondering the same. This seems like a very interesting plugin. Would be very interested in knowing how to customize it. E.g. custom panels, blocks etc. Would it be possible to configure it to output the data as JSON?

**Comment by EsiNakesy** [comment]
Yeah this is very interesting plugging. If we could load the custom blocks that created by filament-fabricator plugging then it would be awesome

**Comment by mohaphez** [comment]
We are working on customizing GrapesJs settings and custom blocks with different outputs including json.

Also, we strongly welcome all those interested in participation.

**Comment by fajriyan** [comment]
I'm looking forward to new block additions and clear documentation, I tried the internet block addition option in this plugin but it doesn't work.
